Checkout updates 2023 (#1548)
carpentries/instructor-training: learners/checkout.md

## Instructor Checkout

After attending Instructor Training, there are three steps to complete before qualifying as a Certified Carpentries Instructor.
Briefly, the three steps are:

1. [Attend a Welcome Session](#welcome). A 1-hour onboarding to the Instructor and global Carpentries community.

> 2. [Complete a Teaching Demonstration](#demonstration). A 1-hour session where trainees demonstrate 5 minutes of teaching and receive and share feedback with their peers.

Author: ```suggestion
2. [Complete a Teaching Demonstration](#demonstration). A 1-hour session where trainees demonstrate 5 minutes of participatory instruction and share feedback, with evaluation by a Carpentries Instructor Trainer.
```

## Instructor Checkout

After attending Instructor Training, there are three steps to complete before qualifying as a Certified Carpentries Instructor.
Briefly, the three steps are:

1. [Attend a Welcome Session](#welcome). A 1-hour onboarding to the Instructor and global Carpentries community.
2. [Complete a Teaching Demonstration](#demonstration). A 1-hour session where trainees demonstrate 5 minutes of teaching and receive and share feedback with their peers.
3. [Get Involved!](#getinvolved). Choose one more step to further your engagement with The Carpentries.

#### How will the Checkout steps be tracked?

> Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts. **For the “Get Involved” step, trainees must login to AMY to report which participation option they have completed.**

Author: ```suggestion
 **For the “Get Involved” step, trainees must login to AMY to report which participation option they would like to have counted.**  Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts.
```

#### Checkout Deadline
All trainees have 3 months (90 days) from the end date of your training to complete checkout. If you need more time,
3-month extensions may be requested by emailing [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org).
Extensions may be granted for any reason up to 1 year from your training date.

:::::::::::::::::::::::::::::::::::::::::  callout
## Help! I have a Question about Checkout! 

> I have you have a question, you can email us at instructor.training@carpentries.org or visit the [#checkout-support](https://swc-slack-invite.herokuapp.com/) slack channel.

Reviewer: ```suggestion
If you have a question, you can email us at instructor.training@carpentries.org or visit the [#checkout-support](https://swc-slack-invite.herokuapp.com/) slack channel.
```

If you arrive at your demo prepared to teach an episode that is ineligible, your Trainer may offer you a choice between
teaching from an appropriate episode (other than the one you prepared) or rescheduling your demo (to allow time to prepare an appropriate episode). 

### Starting in the Middle
Demonstrating an episode that falls later in a lesson can be fun! It also helps to ensure that other trainees who choose the same lesson 
won't all be starting from the same place during the demo session. When selecting an episode that follows others, be aware that some 
steps may rely on files or folders created or modified in earlier episodes. If you choose an episode with dependencies like this, be sure 
to have everything 
in the right place and ready for your start point when you arrive for your demo. 

:::::::::::::::::::::::::::::::::::::::::  callout

## Carpentries Terminology: Lessons vs Episodes
A "lesson" corresponds to a single line in the lesson table and a single repository in GitHub

> (An example of a lesson: [R for Reproducible Scientific Analysis][r-gapminder]). The episodes within a lesson are numbered and listed in a collapsible window on the left side of the main lesson page. After 24 July, 2023, **you do NOT need to prepare a full lesson**, but should select a single episode within that lesson. 

Reviewer: ```suggestion
(An example of a lesson: [R for Reproducible Scientific Analysis][r-gapminder]). The episodes within a lesson are numbered and listed in a collapsible window on the left side of the main lesson page. After 14 August, 2023, **you do NOT need to prepare a full lesson**, but should select a single episode within that lesson. 
```

your Trainer will likely recommend a feedback structure using two axes of positive vs. constructive
and content vs. presentation, just as you have done during this workshop.

### Languages Other than English

If you want to do your demonstration in a language other than English,
please contact us at [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org).
We may not be able to accommodate all languages, but we will do what we can!

### Getting Credit

After each teaching demonstration, Trainers submit a form that includes the names and passing status of each attendee. They will
also email you to let you know, as this information is typically not shared during the demo session. It is not necessary to email us
after you have attended your teaching demonstration.

## 3\. Get Involved! {#getinvolved}

The purpose of this step is to encourage you to take a first step towards further engagement with 
The Carpentries, choosing a path that works well for you. Any participation in Carpentries activities 
or processes may be submitted to satisfy this requirement! We have a few suggestions to get you started.
- Serve as an Instructor or a Helper at a workshop

> - Attend an Instructor Meeting, regional call, or other community programming

Author: ```suggestion
- Attend a regional call, skill-up session, or other community programming
```

- Submit a contribution to a Lesson, Glossario, or other Carpentries repository

### Serve as an Instructor or Helper at a Carpentries Workshop
If you attended Instructor Training because you want to get started participating in workshops as soon 
as possible, this step is for you! When you submit this step, you will be asked to share the date and 
workshop website link. 

Did you serve as a Helper or un-certified co-Instructor before you attended Instructor Training? 
That counts! 

> ### Attend an Instructor Meeting, Regional Call, or Other Community Programming

Author: ```suggestion
### Attend a Regional Call, Skill-Up Session, or Other Community Programming
```

as possible, this step is for you! When you submit this step, you will be asked to share the date and 
workshop website link. 

Did you serve as a Helper or un-certified co-Instructor before you attended Instructor Training? 
That counts! 

### Attend a Regional Call, Skill-Up Session, or Other Community Programming
The Carpentries offers a wide array of programming to meet the needs and interests of our community-members. Attending or hosting any of these 
sessions can be used to fulfill your Get Involved step for Instructor checkout. For more on the different types of Carpentries community 
programming, [see this blog post](https://carpentries.org/blog/2023/04/new-community-programming/) or go directly to the 
[Community Sessions Etherpad][discussion] to look for sessions that will work for you!

### Contribute to the construction or maintenance of Carpentries materials
The Carpentries' lessons are built, sustained, and improved only through the collaborative effort of our community members. 
All Instructors should feel empowered and invited to participate! Taking a first step to walk through the 

> contribution process is a great way to complete your Instructor certification. 

Author: ```suggestion
contribution process is a great way to complete your Instructor certification.  
If you would like help making a lesson contribution consider making a post in the #first-time-contributors channel on [The Carpentries Slack][slack]. This channel is overseen by our Curriculum team and can be useful for contribution questions long after you have completed your checkout process. 
```

[contact-page]: https://carpentries.org/contact/
[carpentries-incubator]: https://github.com/carpentries-incubator/
[glosario-github]: https://github.com/carpentries/glosario/
[discussion]: https://pad.carpentries.org/community-sessions-2023
[welcome]: https://pad.carpentries.org/welcome-sessions-2023
[pad-of-pads]: https://pad.carpentries.org/pad-of-pads
[demos-rubric]: https://carpentries.github.io/instructor-training/demos_rubric
[r-gapminder]: https://swcarpentry.github.io/r-novice-gapminder/

> [start-points]: https://carpentries.github.io/instructor-training/demo_lessons

Author: ```suggestion
[start-points]: https://carpentries.github.io/instructor-training/demo_lessons
[slack]: https://swc-slack-invite.herokuapp.com/
```
